site/docs/components/overlay/examples.mdx

```diff
@@ -0,0 +1,67 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+  <LivePreview componentName="overlay" exampleName="Default" displayName="Default Overlay">
+
+    ### Default overlay
+
+    The basic overlay component allows the user to expose and interact with supplemental information. The overlay remains persistent until the user dismisses it.
+
+    You can configure the `placement` of an overlay to the `top`, `bottom`, `left` or `right` in relation to a UI element.
+
+    #### Best practices
+
+    - Define an `id` and pass the prop `id="YOUR-OVERLAY-ID"` to `Overlay` component.
+    - Add the corresponding prop `id="YOUR-OVERLAY-ID-header"` to your content header or title, and prop `id="YOUR-OVERLAY-ID-content"` to your content description.
```

Comment:
Do we have to repeat this in each example? It seems to me this would belong to Accessibility section

Reply:
I'm inclined to keep it in each example to target users who look only into specific examples and not anywhere else would see the recommended way to use Overlay, happy to keep it in one spot only too if there's strong preference for that @mikearildbrown what do you think?

Reply:
@mikearildbrown updated in this commit https://github.com/jpmorganchase/salt-ds/pull/2860/commits/29864dd10923335a9a95fe926d3e1815e6aaa6d5, please feel free to edit!